deployer/deployer.yaml

```diff
@@ -323,4 +329,16 @@ items:
     description: "(Deprecated) Node selector operations Curator (label=value)."
     name: CURATOR_OPS_NODESELECTOR
     value: ""
+  -
+    description: 'Use the systemd journal for the fluentd log source'
+    name: USE_JOURNAL
+    value: "false"
+  -
+    description: 'The journal source to use when USE_JOURNAL=true'
+    name: JOURNAL_SOURCE
+    value: ""
+  -
+    description: 'Read the journal starting from the very beginning (false|true).'
+    name: JOURNAL_READ_FROM_HEAD
+    value: "false"
```

Comment:
The non-deprecated parameters and env vars in the deployer template are really just to prevent breaking ansible and the instructions (while I work to revamp them). We want to push toward using the deployer configmap and secret for configuration. If new parameters are to be added, they should be marked "(deprecated)" however I think it is best just not to add them at all.